### PR TITLE
airspyhf 1.6.8 (new formula)

### PR DESCRIPTION
**Pull request by drfruct**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by drfruct** [comment]
I don't know how to deal with this error:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 formula detected

It's a library for using Airspy HF SDR receiver, which is very popular among HF radio listeners.

**Comment by nandahkrishna** [comment]
It seems like the repository now meets the notability requirement of 30 forks (it might've been just below the threshold earlier), so I've rerun CI.

**Comment by drfruct** [comment]
Since this is a library for Airspy HF, the only possible test is the version output.

**Comment by SMillerDev** [comment]
> Since this is a library for Airspy HF, the only possible test is the version output.

We have a lot of libraries that get tested by trying to compile a hello world type thing, is that not possible here?

**Comment by drfruct** [comment]
Unfortunately, no, to use the library, you need a physical Airspy HF connection.

**Comment by carlocab** [comment]
Can we try to use a non-existent Airspy HF connection and check we get the expected error?

**Comment by drfruct** [comment]
Yes, that's a good idea.

Added a test that gets the number of connected devices.

**Comment by drfruct** [comment]
> Can we try to use a non-existent Airspy HF connection and check we get the expected error?

Can you check one again?

File: Formula/airspyhf.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  def install
+    args = std_cmake_args
+
+    libusb = Formula["libusb"]
+    args << "-DLIBUSB_INCLUDE_DIR=#{libusb.opt_include}/libusb-1.0"
+    args << "-DLIBUSB_LIBRARIES=#{libusb.opt_lib}/#{shared_library("libusb-1.0")}"
```
Comment (carlocab) [inline Formula/airspyhf.rb:18]:
```suggestion
    args << "-DLIBUSB_INCLUDE_DIR=#{libusb.opt_include}/libusb-#{libusb.version.major_minor}"
    args << "-DLIBUSB_LIBRARIES=#{libusb.opt_lib/shared_library("libusb-#{libusb.version.major_minor}")}"
```

**Review by carlocab — COMMENTED** [review]
The new test looks much better. Just one more minor suggestion.

**Review by carlocab — DISMISSED** [review]
Thanks @drfruct! We can merge this when CI is finished. (I re-ran it to try to build this on Linux too.)

**Review by cho-m — APPROVED** [review]

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**